src/views/Swap/SwapGuru/index.tsx

```diff
@@ -354,42 +356,22 @@ function SwapGuru() {
                     }}
                     IconComponent={() => <SvgIcon component={CaretDownIcon} htmlColor="transparent" />}
                   >
-                    <MenuItem value={"24"} className={classes.menuItem}>
-                      <ListItemText
-                        primary="24 months"
-                        primaryTypographyProps={{
-                          style: {
-                            fontSize: "20px",
-                            lineHeight: "25px",
-                            fontWeight: 400,
-                          },
-                        }}
-                      />
-                    </MenuItem>
-                    <MenuItem value={"12"} className={classes.menuItem}>
-                      <ListItemText
-                        primary="12 months"
-                        primaryTypographyProps={{
-                          style: {
-                            fontSize: "20px",
-                            lineHeight: "25px",
-                            fontWeight: 400,
-                          },
-                        }}
-                      />
-                    </MenuItem>
-                    <MenuItem value={"48"} className={classes.menuItem}>
-                      <ListItemText
-                        primary="48 months"
-                        primaryTypographyProps={{
-                          style: {
-                            fontSize: "20px",
-                            lineHeight: "25px",
-                            fontWeight: 400,
-                          },
-                        }}
-                      />
-                    </MenuItem>
+                    {lockOptions.map((option: any) => {
+                      return (
+                        <MenuItem value={option.value} className={classes.menuItem}>
+                          <ListItemText
+                            primary={option.name}
+                            primaryTypographyProps={{
+                              style: {
```

Comment:
Hmm, could we define with `createStyles` as https://github.com/NidhiDAO/NidhiDAO-frontend/pull/24/files#diff-ecc47e96602cfa104386c81c4b329be4697cf1dfd071fba3bab681a0e69c2103R361 and use it in here